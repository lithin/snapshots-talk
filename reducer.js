const reducer = (state = {}, action) => {
  if (action.type === 'ADD') {
    state.number = 1;
  }
  return state;
}

export default reducer;
