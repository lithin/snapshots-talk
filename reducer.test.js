import reducer from './reducer';

describe('reducer', () => {
  it('returns default state', () => {
    expect(reducer(undefined, {})).toMatchSnapshot();
  });

  it('returns number one for action ADD', () => {
    expect(reducer(undefined, {
      type: 'ADD'
    })).toMatchSnapshot();
  });
});
