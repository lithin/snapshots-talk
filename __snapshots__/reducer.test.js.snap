// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`reducer returns default state 1`] = `Object {}`;

exports[`reducer returns number one for action ADD 1`] = `
Object {
  "number": 1,
}
`;
